web/src/app/components/schedule-drawer.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button } from "@/components/ui/button";
+import {
+  Drawer,
+  DrawerClose,
+  DrawerContent,
+  DrawerDescription,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerTitle,
+  DrawerTrigger,
+} from "@/components/ui/drawer";
+import { Plus } from "@/components/ui/icons";
+
+export default function ScheduleDrawer() {
+  return (
+    <Drawer>
+      <DrawerTrigger>
```

Comment:
```suggestion
      <DrawerTrigger asChild>
```

Essa é a solução pra esse erro que vc tá vendo. O que tá rolando aqui é que esse DrawerTrigger vai criar um componente botão, por padrão. E logo na sequência, vc tá colocando um botão dentro desse botão. Isso não pode acontecer por ser uma violação da estrutura do HTML.

A prop `asChild` faz com que a funcionalidade desse trigger seja adicionada ao botão que está dentro do Drawer eliminando o erro.

Reply:
Fiz a alteração aqui, realmente eliminou o erro.